src/graphics/program-lib/programs/standard.js

```diff
@@ -307,13 +307,25 @@ const standard = {
 
             // specularity & glossiness
             if ((litShader.lighting && options.useSpecular) || litShader.reflections) {
-                const specularPropName = options.useMetalness ? "metalness" : "specular";
                 decl.append("vec3 dSpecularity;");
                 decl.append("float dGlossiness;");
-                code.append(this._addMap(specularPropName, specularPropName + "PS", options, litShader.chunks));
+                if (options.useMetalness) {
```

Comment:
Think you can move this block into the the one below.

Reply:
Yes!